### ISSUE TITLE
Dependency conflicts between imagecodecs and latest libtiff 4.2.0

### ISSUE THREAD

**Issue by rolweber** [ContinuumIO/anaconda-issues#12322]
Hello Team Anaconda,

the latest `libtiff 4.2.0` fixes security issues. It depends on `libwebp-base`, which exists since libwebp 1.1.0. The current verson is libwebp(-base) 1.2.0.

The latest `imagecodecs 2021.1.11` for linux-x86 and linux-ppc64le depends on `libwebp 1.0.x`, and on `libtiff>=4.1.0`. It will therefore be installed with the vulnerable libtiff 4.1.0.

This affects users of scikit-image 0.17.2, via tifffile.

Could you please rebuild imagecodecs with an updated dependency on libwebp 1.2.x?
There's already a build for linux-s390x with that dependency.

**Comment by rolweber** [comment]
There's imagecodecs 2021.3.31 now, but the build for linux-64 still depends on the old libwebp 1.0.x, which pulls in the vulnerable libtiff.

`conda search --platform linux-64 --info imagecodecs=2021.3*=py37*`
```txt
Loading channels: done
imagecodecs 2021.3.31 py37h581e88b_0
------------------------------------
file name   : imagecodecs-2021.3.31-py37h581e88b_0.conda
name        : imagecodecs
version     : 2021.3.31
build       : py37h581e88b_0
build number: 0
size        : 5.9 MB
license     : BSD-3-Clause
subdir      : linux-64
url         : https://repo.anaconda.com/pkgs/main/linux-64/imagecodecs-2021.3.31-py37h581e88b_0.conda
md5         : f9bc4f44e809eef9a56c7aea9842b41c
timestamp   : 2021-04-09 18:40:57 UTC
dependencies: 
...
  - libwebp >=1.0.0,<1.1.0a0
...
```

`conda search --platform linux-s390x --info imagecodecs=2021.*=py37*`
```txt
Loading channels: done
imagecodecs 2021.1.11 py37h0f0a561_2
------------------------------------
file name   : imagecodecs-2021.1.11-py37h0f0a561_2.conda
name        : imagecodecs
version     : 2021.1.11
build       : py37h0f0a561_2
build number: 2
size        : 5.5 MB
license     : BSD-3-Clause
subdir      : linux-s390x
url         : https://repo.anaconda.com/pkgs/main/linux-s390x/imagecodecs-2021.1.11-py37h0f0a561_2.conda
md5         : 612ca4faf2a64853109becc2f12a7710
timestamp   : 2021-02-12 08:10:00 UTC
dependencies: 
...
  - libwebp >=1.2.0,<1.3.0a0
...
```

**Comment by sheetalkalburgi** [comment]
imagecodecs is now available in defaults and dependency issue resolved: imagecodecs 2021.8.26 has libwebp >=1.2.0. Please confirm if this issue is fixed. Closing this issue, please reopen if issue persists.

**Comment by rolweber** [comment]
Thank you. The problem as reported is solved. I can install scikit-image 0.17.2 alongside libtiff 4.2.0.
